lib/cequel/record/persistence.rb

```diff
@@ -260,6 +262,16 @@ def hydrate(row)
         self
       end
 
+      def updater
+        raise ArgumentError, "Can't get updater for new record" if new_record?
+        @updater ||= Metal::Updater.new(metal_scope)
+      end
+
+      def deleter
+        raise ArgumentError, "Can't get deleter for new record" if new_record?
+        @deleter ||= Metal::Deleter.new(metal_scope)
+      end
+
```

Comment:
<img width="1438" alt="screen shot 2018-10-19 at 15 59 59" src="https://user-images.githubusercontent.com/5209518/47238290-21d74780-d3b8-11e8-8ad3-9884b16f1fba.png">

Apparently, these two methods cause lots of warnings because of all the delegations being made unless they're public.

Reply:
That's correct and why I did it. Otherwise, it was a MUCH larger refactor.